Service/EventLogCleanup.php

```diff
@@ -155,9 +185,11 @@ public function deleteEventLogEntries(int $daysOld, ?int $campaignId, bool $dryR
 
             $message .= $result[$operation].' '.$operation;
         }
-
-        $message .= $dryRun ? $this->dryRunMessage : $this->runMessage;
-
+        if (true === $operations['email_stats_tokens']) {
```

Comment:
Use constant key `self::EMAIL_STATS_TOKENS`

Reply:
Thanks @Moongazer for the recommendation to better use constans if we have them - changed this not harmful part in the better practise part with my current commit.